server/package.json

```diff
@@ -26,20 +26,20 @@
     "copy-to-clipboard": "^3.0.8",
     "copy-webpack-plugin": "^4.2.0",
     "crypto": "^1.0.1",
-    "css-loader": "^0.28.7",
+    "css-loader": "^1.0.0",
     "eventsource": "^1.0.5",
     "express": "^4.16.2",
     "express-sslify": "^1.2.0",
-    "extract-text-webpack-plugin": "^3.0.1",
+    "extract-text-webpack-plugin": "^4.0.0-beta.0",
```

Comment:
[`extract-text-webpack-plugin@4.0.0-alpha.0`](https://github.com/webpack-contrib/extract-text-webpack-plugin/releases/tag/v4.0.0-alpha.0) was the first version compatible with Webpack v4. This `beta.0` is the latest prerelease, so maybe it doesn't make sense to fully upgrade until all Webpack v4 related dependencies are stable?